target/linux/ath79/files/drivers/hwmon/rbpoeport.c

```diff
@@ -0,0 +1,311 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
We already have multiple POE drivers in OpenWrt all with different interfaces. It is getting messy.

You could split this and we only take the board support without POE first.